File: thinkup/platform-frontend/src/models/Entity/sponsor.js
class Sponsor extends Student {
  /**
   * Inherits Student Fields and Methods, designed for each Team's Sponsors
   * @param {String} name
   * @param {String} email
   * @param {Settings} settings
   * @param {Permissions} perms
   */

  constructor(name, email, settings, perms) {
    super(name);
    super(email);
    super(settings);
    super(perms);
  }
}

// atributii student +

File: thinkup/platform-backend/src/model/entity/users/sponsor.py
from model.entity.users.student import Student
from model.permissions.permissions import Permissions
from model.settings.settings import Settings


class Sponsor(Student):
    def __init__(self, id: str, name: str, email: str, description: str, settings: Settings, perms: Permissions):
        Student.__init__(id, name, email, description, settings, perms)
